routers/api/packages/npm/npm.go

```diff
@@ -160,6 +160,15 @@ func UploadPackage(ctx *context.Context) {
 		}
 	}
 
+	repository, err := npmPackage.Metadata.Repository.GetGiteaRepository()
```

Comment:
@KN4CK3R do you have some suggestions on how to reliably get the repository of the package?